include/odp/api/spec/packet_io.h

```diff
@@ -121,6 +121,10 @@ typedef union odp_pktin_hash_proto_t {
 		uint32_t ipv6_tcp : 1;
 		/** IPv6 addresses */
 		uint32_t ipv6     : 1;
+		/** Destination MAC address */
+		uint32_t dmac	  : 1;
+		/** Custom L3 fields */
```

Comment:
This would require more explanation. You have some in the commit message but it is needed here. Custom sounds that there is some configuration hidden somewhere  to select the fields. Implementation selecting the fields it wishes sounds more like 'automatic' to me.

Can 'custom' be combined with other fields and what would it mean?